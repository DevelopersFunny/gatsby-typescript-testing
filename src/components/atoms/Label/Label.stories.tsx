import React from 'react'
import { ComponentStory, ComponentMeta } from '@storybook/react'
import { Label } from '.'

export default {
  title: 'Atoms/Label',
  component: Label,
  parameters: {
    fileName: __filename,
  },
  argTypes: {
    backgroundColor: { control: 'color' },
    textColor: { control: 'color' },
  },
} as ComponentMeta<typeof Label>

const Template: ComponentStory<typeof Label> = args => <Label {...args} />

export const LabelName = Template.bind({})
LabelName.args = {
  children: 'Label',
}
