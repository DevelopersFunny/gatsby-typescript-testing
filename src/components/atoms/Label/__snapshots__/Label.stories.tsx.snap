// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stories storyshots Atoms/Label Label Name 1`] = `
<label
  className="css-72egtx"
  style={
    Object {
      "backgroundColor": undefined,
      "color": undefined,
    }
  }
>
  Label
</label>
`;
